modules/ROOT/pages/major-changes.adoc

```diff
@@ -9,6 +9,40 @@ This list is in reverse chronological order to keep recent changes at the top.
 
 // To add a new change here, see the template at the end of the file.
 
+== Switch to OCI updates
+
+Starting in Fedora 42, Fedora CoreOS will be updating through OCI images instead of the
+OSTree repo.
+Fedora Change request: https://fedoraproject.org/wiki/Changes/CoreOSOstree2OCIUpdates
+More discussion: https://github.com/coreos/fedora-coreos-tracker/issues/1823
+
+=== Planning
+
+This change has been introduced first in bootimages for the testing stream, for the Fedora 42 rebase. After a few releases
+we will migrate existing machines through a barrier release.
+
+|===
+|Update Stream |Targeted release date
+
+|`next` | 2025-02-04
+|`testing`| 2025-04-22
+|`stable`| Will follow `testing` as usual
+|===
+
+=== Notes
+
+Currently, Fedora CoreOS hosts pull updates from the OSTree repository.
+With this change, the hosts will pull updates from the Quay.io container registry instead.
+This is a transparent change, except for proxied environnements, as the nodes will reach to a different address for updates.
```

Comment:
We claim that it's transparent and then mention exceptions. Let's say that it should be transparent but the following points may require attention: proxy configuration, etc.

Reply:
reworded